elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -143,13 +143,22 @@ export abstract class BaseTransaction {
 		const tx = (typeof rawTransaction === 'object' && rawTransaction !== null
 			? rawTransaction
 			: {}) as Partial<TransactionJSON>;
+
 		this.amount = new BigNum(
 			isValidNumber(tx.amount) ? (tx.amount as string | number) : '0',
 		);
+
 		this.fee = new BigNum(
-			isValidNumber(tx.fee) ? (tx.fee as string | number) : '0',
+			isValidNumber(tx.fee)
+				? (tx.fee as string | number)
+				: (this.constructor as typeof BaseTransaction).FEE,
 		);
 
+		this.type =
+			typeof tx.type === 'number'
```

Comment:
Maybe not to be fixed in this PR but should we discuss a way to let the user know we're defaulting to something or maybe create an issue to document this behaviour?